browser/model/cdk.js

```diff
@@ -1,15 +1,15 @@
 'use strict';
 
 let fs = require('fs-extra');
-let request = require('request');
 let path = require('path');
-let unzip = require('unzip');
 let ipcRenderer = require('electron').ipcRenderer;
+let Promise = require('bluebird');
```

Comment:
@jrichter1 latest version of Electron contains Node.js 4.1.1 which already implements Promise so Bluebird is not necessary. It should be removed both in code and in package.json